xline/src/server/maintenance.rs

```diff
@@ -0,0 +1,225 @@
+use std::sync::Arc;
+
+use clippy_utilities::{Cast, OverflowArithmetic};
+use sha2::{Digest, Sha256};
+use tokio::sync::mpsc;
+use tokio_stream::wrappers::ReceiverStream;
+use tracing::error;
+
+use crate::{
+    header_gen::HeaderGenerator,
+    rpc::{
+        AlarmRequest, AlarmResponse, DefragmentRequest, DefragmentResponse, DowngradeRequest,
+        DowngradeResponse, HashKvRequest, HashKvResponse, HashRequest, HashResponse, Maintenance,
+        MoveLeaderRequest, MoveLeaderResponse, SnapshotRequest, SnapshotResponse, StatusRequest,
+        StatusResponse,
+    },
+    storage::storage_api::StorageApi,
+};
+
+/// Snapshot chunk size
+const SNAPSHOT_CHUNK_SIZE: u64 = 64 * 1024;
```

Comment:
I think we can define these two const values separately, because they are used in different grpc requests and may not be exactly the same

Reply:
If curp changes the `SNAPSHOT_CHUNK_SIZE` to another value, like 32KB or 128KB, which is different from the `SNAPSHOT_CHUNK_SIZE` defined here, will it introduce some inconsistent behavior? If so, we should use the unique `SNAPSHOT_CHUNK_SIZE`.  If not, we'd better tell them apart by different names, at least it will decrease some confusion when someone reads the code.

Reply:
Renamed, it does not introduce inconsistent behavior